examples/step-54/doc/intro.dox

```diff
@@ -79,7 +79,7 @@ onto the specified geometry.
 
 Differently from a spherical or circular boundary, a boundary with a complex
 geometry poses problems as to where it is best to place the new nodes created
-upon refinement on the prescribed shape.  HyperBallBoundary first creates the
+upon refinement on the prescribed shape. SphericalManifold first creates the
 new nodes on the face or edge to be refined by averaging the surrounding
 points in the same way as FlatManifold does. Then, it goes on to project such
```

Comment:
> new nodes on the face or edge to be refined by averaging the surrounding points in the same way as FlatManifold does.

I don't think that this is what `SphericalManifold` does, but I might be wrong (I think the averaging is not the same as what is done by `FlatManifold`).

Reply:
You are totally right. I switched to `PolarManifold` which is much easier to describe.